source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.
+
+Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).
+
+## Action Required
+
+To receive security ongoing security coverage for your contrib modules, apply this upstream update then enable the Tag1 D7ES module.
+
```

Comment:
In our [internal notes](https://docs.google.com/document/d/17Fs08vSLoRShdvPFGzRz_KCuY9rFke8QA3-3KTMNn4s/edit?tab=t.0) we also called out the need/suggestion of configuring email alerts

Reply:
That's referenced in the Tag1 documentation (which we link to). Our docs do not talk about setting up email alerts. We should add that as a heading to the docs page and just link to https://docs.tag1.com/notifications/

Reply:
We should replace the Tag1 documentation with our own. The Tag1 version instructs the user to do things not necessary on Pantheon. What's more, it asks the user to look up information in the module that is not displayed when on Pantheon, which could cause confusion for users.

Reply:
Our docs do talk about configuring email alerts. I added a screenshot and a paragraph about notifications.